tensorflow/python/keras/layers/convolutional_recurrent.py

```diff
@@ -20,6 +20,7 @@
 from __future__ import print_function
 
 import numpy as np
+from six.moves import zip  # pylint: disable=redefined-builtin
```

Comment:
why is this needed?

Reply:
The behaviour of `zip` changed in Python 3. To ensure the same behaviour, in Python 2 it will use `itertools.izip()` where as in Python 3 it will use `zip()`. For more info see: https://six.readthedocs.io/#module-six.moves

Reply:
If it is fine that `zip` returns a `list` in Python 2 and an `iterator` in Python 3, I can switch to using the builtin `zip` here.

Reply:
@tanzhenyu With TensorFlow slowly dropping support for Python 2, this minor detail doesn't matter anymore, so I removed the import from the PR.